### ISSUE TITLE
Interlink terms in UJS with the glossary

### ISSUE THREAD

**Issue by Julian** [json-schema-org/understanding-json-schema#196]
It seems these two resources would benefit a lot from each other, [the latter](https://json-schema.org/learn/glossary.html) as a reference for terms used in UJS, and UJS obviously as an expository resource to link to from the glossary.

Assuming others agree, I wrote a [Sphinx plugin](https://sphinx-json-schema-spec.readthedocs.io/en/latest/) which this repository could use via [intersphinx](https://docs.readthedocs.io/en/stable/guides/intersphinx.html) and am happy to send a PR adding some initial interlinks.

CC @jdesrosiers since I see it's mostly you helping move this repo along lately :)

**Comment by jdesrosiers** [comment]
I agree that the two should be interlinked, but considering that UJS is expected to be incorporated into the the new website, I think any updates that involve a Sphinx plugin will just get lost in the transition. Maybe it's best to wait to make a change like this until it's on the new platform.
